wemake_python_styleguide/violations/complexity.py

```diff
@@ -572,6 +574,50 @@ class TooManyAssertsViolation(ASTViolation):
     code = 218
 
 
+@final
+class TooDeepAccessViolation(ASTViolation):
+    """
+    Forbids to have consecutive expressions with too deep access level.
+
+    We consider these expressions as accesses:
+
+        ``ast.Subscript``
```

Comment:
Should be a list:

```
- ``ast.Subscript``
- ``ast.Attribute``
```

Please, also mention that ``ast.Call` is treated differently (and why so).